phpunit/tests/import.php

```diff
@@ -20,29 +34,37 @@ function setUp() {
 		add_filter( 'import_allow_create_users', '__return_true' );
 
 		global $wpdb;
-		// crude but effective: make sure there's no residual data in the main tables
+		// Crude but effective: make sure there's no residual data in the main tables.
 		foreach ( array( 'posts', 'postmeta', 'comments', 'terms', 'term_taxonomy', 'term_relationships', 'users', 'usermeta' ) as $table ) {
-			$wpdb->query( "DELETE FROM {$wpdb->$table}" );
+			$wpdb->query( "DELETE FROM {$wpdb->$table}" ); // phpcs:ignore
```

Comment:
Should this ignore be made more specific ? I.e. just ignore one or more sniffs ? or a sniff category ?
Now it's a blanket ignore for everything and will also not report on new issues being introduced into the line.

Reply:
It was a quick way to ignore multiple issues. More specific fixings could be done once the mass of issues are done.